css/bulma-rtl.css

```diff
@@ -6839,7 +6817,7 @@ label.panel-block:hover {
 
 .tabs.is-boxed a {
   border: 1px solid transparent;
-  border-radius: 0 0 4px 4px;
```

Comment:
There was a bug during previous rtl conversion since non rtl it is 4px 4px 0 0, it should have been 4px 0 0 4px (left and right are reversed, but currently top and bottom also got reversed)